src/index.html

```diff
@@ -1,13 +1,43 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="product-card" data-qa="card">
+      <section class="product-card__picture"></section>
+
+      <arlicle class="product-card__description-text ">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</arlicle>
+      <details class="product-card__description-code">Код товара: 195434</details>
+
+      <section class="product-card-rating">
+        <div class="stars stars--4">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+        </div>
+        <div class="product-card-rating__count-review">
+          Отзывов: 5
+        </div>
+      </section>
+
+      <section class="product-card__prise">
+        <div class="product-card__prise-text">Цена:</div>
+        <div class="product-card__prise-amount">69 999 грн</div>
+      </section>
+
+      <a href="#bye" data-qa="hover" class="product-card__bye">
+        <input type="button" value="купить" class="product-card__bye-button">
+      </a>
```

Comment:
Check your code style according to [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and fix your tags